lib/core/src/lib/styles/_mat-selectors.scss

```diff
@@ -43,11 +43,13 @@ $mat-progress-bar-buffer: '.mdc-linear-progress__buffer';
 $mat-progress-spinner: '.mat-mdc-progress-spinner';
 $mat-form-field: '.mat-mdc-form-field';
 $mat-form-field-flex: '.mat-mdc-form-field-flex';
+$mat-form-field-outline: '.mat-mdc-form-field-outline';
 $mat-form-field-wrapper: '.mat-mdc-text-field-wrapper';
 $mat-line-ripple: '.mdc-line-ripple';
 $mat-form-field-subscript-wrapper: '.mat-mdc-form-field-subscript-wrapper';
 $mat-form-field-prefix: '.mat-mdc-form-field-text-prefix';
 $mat-form-field-suffix: '.mat-mdc-form-field-text-suffix';
+/* stylelint-disable-next-line scss/dollar-variable-pattern */
```

Comment:
These selectors have already been [updated](https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/core/src/lib/styles/_mat-selectors.scss#L51) in develop to no longer fail the stylelint rule. If you get the latest changes via rebase, then you should no longer need to disable the rule.

Reply:
I will rebase main feature 7600 branch and it will be updated